src/main_chart.py

```diff
@@ -93,20 +93,16 @@ def get_main_chart_figure(
 ) -> plotly.graph_objs.Figure:
     # Define the AMMs and their color mappings
     amms = src.swap_amm.AMMS
-    color_map = {
-        "10kSwap_debt_token_supply": "#4C6DD0",
-        "MySwap_debt_token_supply": "#4CA7D0",
-        "SithSwap_debt_token_supply": "#4C99D0",
-        "JediSwap_debt_token_supply": "#4C83D0",
-    }
+    color_map = {"10kSwap_debt_token_supply": "#1f77b4",
+                 "MySwap_debt_token_supply": "#ff7f0e",
+                 "SithSwap_debt_token_supply": "#2ca02c",
+                 "JediSwap_debt_token_supply": "#d62728"}
 
     # TODO: Align colors with the rest of the app.
-    color_map_protocol = {
-        "liquidable_debt_at_interval_zkLend": "#fff7bc",  # light yellow
-        "liquidable_debt_at_interval_Nostra Alpha": "#fec44f",  # yellow
-        "liquidable_debt_at_interval_Nostra Mainnet": "#d95f0e",
-    }  # mustard yellow
-    color_map_liquidity = {"debt_token_supply": "#1f77b4"}  # blue
+    color_map_protocol = {"liquidable_debt_at_interval_zkLend": "#4B16E9", 
```

Comment:
Move it to constant

Reply:
Do you mean define them in all uppercase to follow Python's convention for constants

Reply:
or to make the dictionary global

Reply:
Moving it to constant, with uppercase is a good idea to have all colors in one place. 
It will be easier to manage colors in the future